.vscode/launch.json

```diff
@@ -13,7 +13,7 @@
       "args": [
         "--opts",
         "${workspaceRoot}/test/mocha.opts",
-        "packages/*/test/**/*.ts",
+        "packages/*/dist/test/**/*.js",
```

Comment:
We still support Node 6 (till that's dropped), idk if switching to just `dist` is a good idea.

Reply:
We can have multiple configuration profiles if necessary. I'm fine to have only `dist` though.

Reply:
Here is my take:

 - The current wildcard "packages/*/test/**/*.ts" does not work at all, so any change is for better
 - While we do support Node.js 6.x for people using our framework, I don't think we have to necessarily support Node.js 6.x as the primary developer environment. 
 - For the rare occasions when we need to debug specifically on Node.js 6.x, we can either temporarily change `dist` to `dist6`, or add a new configuration profile as suggested by Raymond above.